packages/protect/src/ffi/index.ts

```diff
@@ -194,11 +201,13 @@ class EncryptOperationWithLockContext
         const val = await ffiEncrypt(
           client,
           plaintext,
-          column,
+          column.getName(),
+          table.tableName,
```

Comment:
Did you get a chance to re-test lock contexts with the searchable encryption changes?

Reply:
Yes it's all sorted